app/assets/sass/plugins/bootstrap-datetimepicker/bootstrap-datetimepicker.scss

```diff
@@ -0,0 +1,303 @@
+        }
+
+        &.today {
+            position: relative;
+
+            &:before {
+                content: '';
+                display: inline-block;
+                border-left: 7px solid transparent;
+                border-bottom: 7px solid $btn-primary-bg;
+                border-top-color: rgba(0, 0, 0, 0.2);
+                position: absolute;
+                bottom: 4px;
+                right: 4px;
+            }
+        }
+
+        &.active,
+        &.active:hover {
+            background-color: $btn-primary-bg;
+            color: $btn-primary-color;
+            text-shadow: 0 -1px 0 rgba(0,0,0,.25);
+        }
+
+        &.active.today:before {
+            border-bottom-color: #fff;
+        }
+
+        &.disabled,
+        &.disabled:hover {
+            background: none;
+            color: $gray-light;
+            cursor: not-allowed;
+        }
+
+        span {
+            display: inline-block;
+            width: 54px;
+            height: 54px;
+            line-height: 54px;
+            margin: 2px 1.5px;
+            cursor: pointer;
+            border-radius: $border-radius-base;
+
+            &:hover {
+                background: $gray-lighter;
+            }
+
+            &.active {
+                background-color: $btn-primary-bg;
+                color: $btn-primary-color;
+                text-shadow: 0 -1px 0 rgba(0,0,0,.25);
+            }
+
+            &.old {
+                color: $gray-light;
+            }
+
+            &.disabled,
+            &.disabled:hover {
+                background: none;
+                color: $gray-light;
+                cursor: not-allowed;
+            }
+        }
+    }
+
+    th {
+        height: 20px;
+        line-height: 20px;
+        width: 20px;
+
+        &.picker-switch {
+            width: 145px;
+        }
+
+        &.next,
+        &.prev {
+            font-size: $font-size-base * 1.5;
+        }
+
+        &.disabled,
+        &.disabled:hover {
+            background: none;
+            color: $gray-light;
+            cursor: not-allowed;
+        }
+    }
+
+    thead tr:first-child th {
+        cursor: pointer;
+
+        &:hover {
+            background: $gray-lighter;
+        }
+    }
+}
+
+.input-group {
+    &.date {
+        .input-group-addon span {
+            display: block;
+            cursor: pointer;
+            width: 16px;
+            height: 16px;
+        }
+    }
+}
+
+.bootstrap-datetimepicker-widget.left-oriented {
+    &:before {
+        left: auto;
+        right: 6px;
+    }
+
+    &:after {
+        left: auto;
+        right: 7px;
+    }
+}
+
+.bootstrap-datetimepicker-widget ul.list-unstyled li div.timepicker div.timepicker-picker table.table-condensed tbody > tr > td {
+    padding: 0px !important;
+}
+
+@media screen and (max-width: 767px) {
+    .bootstrap-datetimepicker-widget.timepicker-sbs {
+        width: 283px;
+    }
+}
+
+.sr-only {
+  position: absolute;
```

Comment:
I copied the original code from the `bower_components` directory as I couldn't get it to compile, I didn't even think about the formatting :(